core/test/unit/storage/index_spec.js

```diff
@@ -37,6 +37,11 @@ describe('storage: index_spec', function () {
             (chosenStorage instanceof localFileStorage).should.eql(true);
         });
 
+        it('escape non accepted characters in filenames', function () {
```

Comment:
could you please move this test into a test file named `storage/base_spec.js`?
the test you have added is now part of a test section which covers loading storage adapters.

and if you have time, the logic you have added is not covered by a test. you could copy this test https://github.com/TryGhost/Ghost/blob/master/core/test/functional/routes/api/themes_spec.js#L73 and just upload a different theme with some special characters.
thanks 👍